src/components/ui/Separator/stories/Separator.stories.tsx

```
@@ -1,12 +1,14 @@
-import Separator from '../Separator';
+import { JSX, ClassAttributes, HTMLAttributes } from 'react';
```

ESLint

[error] '/home/jailuser/git/node_modules/react/index.js' imported multiple times.

(import/no-duplicates)